js/ui/grid_core/ui.grid_core.modules.js

```diff
@@ -439,5 +439,7 @@ export default {
 
     processModules: processModules,
 
-    callModuleItemsMethod: callModuleItemsMethod
+    callModuleItemsMethod: callModuleItemsMethod,
+
+    VIEW_NAMES: ['columnsSeparatorView', 'blockSeparatorView', 'trackerView', 'headerPanel', 'columnHeadersView', 'rowsView', 'footerView', 'columnChooserView', 'filterPanelView', 'pagerView', 'draggingHeaderView', 'contextMenuView', 'errorView', 'headerFilterView', 'filterBuilderView']
```

Comment:
Bad place for this array. The grid_core.modules does not have code that are related to specific widget. Export it from grid_view.js